utils/is-audited.js

```diff
@@ -1,15 +1,25 @@
 // this can go once all certs have been audited.
+
+// Currently the auditing is going through Crowdin, so once a cert has been 100%
+// proofread, we can add it in here. That means that translations can come
+// through from Crowdin whenever they are done, but we don't show them on the
+// client until we decide the entire cert is ready.
+
+// NOTE: certificates themselves (.markdown files) are not currently being
+// translated, but when they are they can be included by adding 'certificates'
+// to the arrays below
+
+const auditedCerts = {
+  espanol: [],
+  chinese: []
+};
+
 function isAuditedCert(lang, cert) {
   if (!lang || !cert)
     throw Error('Both arguments must be provided for auditing');
   // in order to see the challenges in the client, add the certification that
   // contains those challenges to this array:
```

Comment:
Leys drop these comments - classic example of how comments go stale. I much rather not comment vs letting them go stale.

Reply:
I get your point, but code can go stale, too.

I'll do some reading around the topic, but I feel like the cost of having to catch this in QA (like this) is outweighed by the benefit that the comment provides.

Reply:
Lines 20-21 as [seen here](https://github.com/freeCodeCamp/freeCodeCamp/commit/6269f783cc7291b75af3aac6e62890f1bac81e33#diff-3a4b75b56b44645bb3c05d7031cf913278c9efd2fe4d78a1f6518555ac3dbe26R20-R21) are now stale. I am not talking about lines 3-10 as seen here [https://github.com/freeCodeCamp/freeCodeCamp/commit/6269f783cc7291b75af3aac6e62890f1bac81e33#diff-3a4b75b56b44645bb3c05d7031cf913278c9efd2fe4d78a1f6518555ac3dbe26R3-R10]

Reply:
Yeah, I got that. I agree that these stale ones should go. It's possible were talking about different things - I thought you were arguing that comments than *can* go stale are bad.

Reply:
Let's agree to disagree. 

The point being, at the end of the day as a maintainer - it's us who are put at blame by the regardless of how much efforts you put in to make lives of the contributors easier. 

I do not care for comments in code, others do - and that's okay. There are well established patterns in out there to maintain code properly, I personally do not have had the time because this is bike shedding territory. Any documentation that we do have has been very painstakingly done (with significant contributions from me too. So, I get why we should have code comments).

The only thing that I do care about is the unfathomable tech-debt, and the code quality. Those are my highest priorities over code comments and documentation.

Reply:
Understood.  I think this is a conversation worth having, but not now and not buried away here.  Let's pick this up again when we both have a bit more time.